package.json

```diff
@@ -2,12 +2,12 @@
   "name": "@react-native-picker/picker",
   "version": "1.16.6",
   "homepage": "https://github.com/react-native-picker/picker#readme",
-  "description": "React Native Picker for iOS & Android",
+  "description": "React Native Picker for iOS, Android, macOS, and Windows",
   "main": "./js/index.js",
   "types": "./typings/index.d.ts",
   "scripts": {
     "start": "react-native start",
-    "start:windows": "react-native start --projectRoot ./ --use-react-native-windows",
```

Comment:
Don't we need a start Windows?

Reply:
Normal `start` works now. This special one was to use a Windows custom Metro config which is no longer needed with newer versions of RN.